### PR TITLE
Feature/taipy 2267 fixwindows linux paths

### PR DESCRIPTION
**Pull request by jrobinAV**
This will be merged when hotfix 4.0.2 is released.

Two new entries in the release notes:
- https://github.com/Avaiga/taipy/issues/2319
- https://github.com/Avaiga/taipy/issues/2261

### CODE REVIEW

File: docs/release-notes/index.md [hunk 1]
```diff
@@ -272,12 +272,20 @@ following new functionalities:
 
 ## Significant bug fixes
 
+<h4><strong><code>taipy-core</code></strong> 4.0.2</h4>
+- Some data node update events were not correctly published on the notifier.<br/>
+  See [issue #2319]https://github.com/Avaiga/taipy/issues/2319
+- Running a Taipy application on a unix base Docker container with file data nodes (csv, excel,
```
Comment (FabienLelaquais) [inline docs/release-notes/index.md]:
```suggestion
- Running a Taipy application on a Unix-based Docker container with file data nodes (csv, excel,
```

**Review by FabienLelaquais — APPROVED** [review]
Not too sure why this is for release/4.0 and not develop, though

File: docs/release-notes/index.md [hunk 2]
```diff
@@ -272,12 +272,20 @@ following new functionalities:
 
 ## Significant bug fixes
 
+<h4><strong><code>taipy-core</code></strong> 4.0.2</h4>
+- Some data node update events were not correctly published on the notifier.<br/>
+  See [issue #2319]https://github.com/Avaiga/taipy/issues/2319
```
Comment (jrobinAV) [inline docs/release-notes/index.md]:
```suggestion
  See [issue #2319](https://github.com/Avaiga/taipy/issues/2319).
```

**Review by FabienLelaquais — APPROVED** [review]
👌

**Pull request merged**